app/scripts/analysis/tools/powercalc/PowercalcMainCtrl.ctrl.coffee

```diff
@@ -0,0 +1,1529 @@
+'use strict'
```

Comment:
this is a huge monster file – meeds to be broken down into separate services. possibly per individual calculator?